WebRoot/css/zm.css

```diff
@@ -6669,7 +6669,7 @@ a.ZmLinkDisabled {
 
 .ZDialogButton .ZButtonTable .ZWidgetTitle {
     color: @AltC@;
-    letter-spacing: 1px;
+    font-size:1.1rem;
```

Comment:
@FontSize-slightly-big@

Reply:
changing it to use the property. Not moving these styles to a property variables as I think there would be more efforts moving all dialog related stylings into variable types.

Reply:
Fixed here:
3f69c82ea0a80ec5e23f8f79e037f97225ba4c43